notebooks-auth/test_thredds.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>All the <code>assert</code>  in this cells seems to be identical to the previous cell, copy/paste error again?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>